upload-api/tables/index.js

```diff
@@ -91,6 +74,7 @@ export const consumerTableProps = {
   // TODO does this index setup seem right?
   primaryIndex: { partitionKey: 'consumer', sortKey: 'subscription' },
   globalIndexes: {
+    // TODO: I don't think we should keep this index - partitioning by provider is basically useless and won't be much faster than a table scan
```

Comment:
could we get a small markdown doc with all tables schemas and queries we need to do so that we can align better on which indexes we should use?

Reply:
so what I've been working toward is a test that demonstrates the various queries we need to do - I've implemented some of them here but need to do a bit more prototyping/research to be more certain about the set of queries we need to do:

https://github.com/web3-storage/w3infra/pull/194/files#diff-924a94ba76534a93c7fa4c89f8e70037f21694b6374beaca7a2c6ebf3fdb2006

does this give you the info you're looking for? I'd rather work in code at the moment since this is changing fairly quickly but can definitely compile info from this file and the queries tests into one doc if that would help a lot!